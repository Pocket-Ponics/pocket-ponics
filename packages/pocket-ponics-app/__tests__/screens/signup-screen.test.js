import React from 'react'
import { shallow } from 'enzyme'

import SignUpScreen from '../../screens/signup-screen'

describe('Sign Up Screen', () => {
	test('renders correctly', () => {
		const tree = shallow(
			<SignUpScreen />
		).html()
		expect(tree).toMatchSnapshot()
	})
})