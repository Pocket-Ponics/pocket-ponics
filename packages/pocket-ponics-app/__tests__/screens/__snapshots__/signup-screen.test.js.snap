// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sign Up Screen renders correctly 1`] = `"<View style=\\"0:[object Object];1:[object Object]\\"><View style=\\"flex:1;align-items:center;justify-content:center;padding:20px;margin-top:40px;width:100%\\"><Image source=\\"1\\" style=\\"height:200px;width:200px;resize-mode:contain\\"></Image><View style=\\"margin-top:5px;align-items:center;width:100%\\"><Text></Text><TextInput underlineColorAndroid=\\"transparent\\" style=\\"width:100%;height:45px;border-radius:45px;font-size:16px;padding-left:45px;background-color:rgba(0, 0, 0, 0.35);color:#FFFFFF\\" placeholder=\\"Email Address\\" placeholderTextColor=\\"rgba(255, 255, 255, 0.7)\\" value=\\"\\" autoCapitalize=\\"none\\" textContentType=\\"emailAddress\\"></TextInput></View><View style=\\"margin-top:5px;align-items:center;width:100%\\"><Text></Text><TextInput underlineColorAndroid=\\"transparent\\" style=\\"width:100%;height:45px;border-radius:45px;font-size:16px;padding-left:45px;background-color:rgba(0, 0, 0, 0.35);color:#FFFFFF\\" placeholder=\\"Password\\" placeholderTextColor=\\"rgba(255, 255, 255, 0.7)\\" value=\\"\\" autoCapitalize=\\"none\\" textContentType=\\"newPassword\\"></TextInput></View><View style=\\"margin-top:5px;align-items:center;width:100%\\"><Text></Text><TextInput underlineColorAndroid=\\"transparent\\" style=\\"width:100%;height:45px;border-radius:45px;font-size:16px;padding-left:45px;background-color:rgba(0, 0, 0, 0.35);color:#FFFFFF\\" placeholder=\\"Verify Password\\" placeholderTextColor=\\"rgba(255, 255, 255, 0.7)\\" value=\\"\\" autoCapitalize=\\"none\\" textContentType=\\"newPassword\\"></TextInput></View><View style=\\"opacity:0.3\\"><Text style=\\"background-color:#456E31;border-radius:5px;padding:10px;margin:10px;width:100%;align-items:center\\">Sign Up</Text></View><View style=\\"opacity:1\\"><Text style=\\"color:#84AB71;font-size:18px;margin-bottom:10px\\">Log In</Text></View><ActivityIndicator color=\\"#FFFFFF\\"></ActivityIndicator></View></View>"`;
